RULES.md

```diff
@@ -25,7 +25,7 @@ Checks that the mapped name of a field is the expected snake case.
 ```ts
 z.object({
   compoundWords: z.array(z.string()).optional(),
-  requirePrefix: z.string().optional(),
```

Comment:
should this be kept and marked as deprecated instead of making a breaking change?